api/bus.go

```diff
@@ -9,6 +9,15 @@ import (
 	"go.sia.tech/renterd/object"
 )
 
+var (
+	// ErrHostNotFound is returned if the requested host is not found.
+	ErrHostNotFound = errors.New("host doesn't exist in hostdb")
+
+	// ErrSettingNotFound is returned if a requested setting is not present in the
+	// database.
+	ErrSettingNotFound = errors.New("setting not found")
```

Comment:
Is there a reason for moving these errors to the `API` package? Seems like they aren't used in any additional places now?

Reply:
Hm I think I just moved it because `ErrHostNotFound` should probably be in the `API` package. One of the drivers for the `api` package was exported-error-related. I'll move it back though I don't mind at all.